### PR TITLE
Update ReadMe to use GitHub formatting

### PR DESCRIPTION
**Pull request by adamrusted**
This PR serves to use the new [GitHub Markdown](https://github.com/orgs/community/discussions/16925) highlighting of notes/warnings in our main README file.
Updated the 'strip links' script to also revert these notes/warnings to our previous style when shared elsewhere.

### CODE REVIEW

File: scripts/release/strip-theme-links.js [hunk 1]
```diff
@@ -17,8 +17,17 @@ const readmeFile = path.resolve(rootDir, 'README.md');
...
+      `https://raw.githubusercontent.com/simple-icons/simple-icons/${LINKS_BRANCH}/icons/$1.svg`,
+    )
+    .replace(
+      /\[!(NOTE|TIP|IMPORTANT|WARNING|CAUTION)\](?!\()/g,
+      function (str, $0) {
+        const capital = $0.substr(0, 1);
+        const body = $0.substr(1).toLowerCase();
+        return `**${capital + body}**`;
+      },
+    ),
```
Comment (LitoMore) [inline scripts/release/strip-theme-links.js:32]: This `strip-theme-links.js` script is only used for strip theme links.

Comment (adamrusted) [reply scripts/release/strip-theme-links.js]: Have updated the PR to reflect file renaming. Thanks @LitoMore!

**Review by mondeja — APPROVED** [review]
Looks good to me.

I personally think that this change is technically a bad movement from GitHub. They are just totally breaking READMEs in other services that pull from GH. But I guess that MS wants money from integrations, it's fair 🤑 

For example, in Packagist the current approach at least looks aceptable.
image


The rendering of my formatter for VSCode becomes really ugly:
image

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Over 2700 Free SVG icons for popular brands. See them all on one page at <a href
 
 ## Usage
 
-> **Note**\
+> [!NOTE]\
```
Comment (mondeja) [inline README.md:20]:
```suggestion
> [!NOTE]
```

File: README.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 | [Vue package](https://github.com/mainvest/vue-simple-icons) <img src="https://cdn.simpleicons.org/vuedotjs/000/fff" alt="Vue" align=left width=24 height=24> | [@noahlitvin](https://github.com/noahlitvin) | ![](https://img.shields.io/github/license/mainvest/vue-simple-icons) | [![v4.7.0](https://img.shields.io/static/v1?label=version&message=v4.7.0&logo=simpleicons&color=red)](https://github.com/simple-icons/simple-icons/tree/4.7.0) |
 | [WordPress plugin](https://wordpress.org/plugins/simple-icons/) <img src="https://cdn.simpleicons.org/wordpress/000/fff" alt="WordPress" align=left width=24 height=24> | [@tjtaylo](https://github.com/tjtaylo) | ![](https://img.shields.io/badge/license-GPL_v2-blue.svg) | [![v4.25.0](https://img.shields.io/static/v1?label=version&message=v4.25.0&logo=simpleicons&color=red)](https://github.com/simple-icons/simple-icons/tree/4.25.0) |
 
-> **Note for Extension Authors**\
+> [!IMPORTANT]\
```
Comment (mondeja) [inline README.md:182]:
```suggestion
> [!IMPORTANT]
```

**Review by mondeja — CHANGES_REQUESTED** [review]
Wait a moment, The `\` characters must be removed, right?

**Review by mondeja — APPROVED** [review]

**Comment by adamrusted** [comment]
> The rendering of my formatter for VSCode becomes really ugly

Yeah, same. Hopefully they fix it. The packagist approach is what we revert to when publishing, so hopefully doesn't break anything!

**Comment by adamrusted** [comment]
No worries @mondeja - I'll update the PR today to add the newline back in when reformatting for npm/Packagist

**Review by PratyushJayachandran — APPROVED** [review]

**Review by mondeja — APPROVED** [review]
> but I want to see if I [hear back](https://discord.com/channels/308323056592486420/477266055169835030/1178980270632869909) from the `shields.io` team before we merge this

Please use another pull request. It is better to have the changes separated so as not to get confused when checking past changes.

**Pull request merged**